cross/shine/Makefile

```diff
@@ -0,0 +1,35 @@
+PKG_NAME = shine
+PKG_VERS = 3.1.1
+PKG_EXT = tar.gz
+PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_NAME = $(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/toots/shine/archive
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = 
+
+HOMEPAGE = https://github.com/toots/shine
+COMMENT  = Super fast fixed-point MP3 encoder
+LICENSE  = GPLv2
+
+PRE_CONFIGURE_TARGET = shine_pre_configure
+#POST_INSTALL_TARGET = shine_post_install
```

Comment:
Please discard commented parts for merge. You may keep it in a local branch or notes if it may be useful later.

Reply:
done.